sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -492,16 +501,16 @@ export class SecretsClient {
    * @param [options] The optional parameters
    * @returns Promise<Uint8Array | undefined>
    */
-  public async backupSecret(secretName: string, options?: RequestOptionsBase): Promise<Uint8Array> {
+  public async backupSecret(secretName: string, options?: RequestOptionsBase): Promise<Uint8Array | undefined> {
```

Comment:
Oh interesting that this was already documented as potentially resolving with `undefined`.

Reply:
Yeah it seemed like a mistake was made on the typing of the interface